From e9a6ebffbb5cdd3d3f452610bda4371e51288058 Mon Sep 17 00:00:00 2001
From: Xe Iaso <me@xeiaso.net>
Date: Fri, 28 Mar 2025 07:39:14 -0400
Subject: [PATCH] data: disable DroneBL lookups by default (#147)

Closes #109

This was a hack I did on stream. I thought this would have a positive
effect, but a combination of real-world testing from people using Anubis
in prod and gray-hat testing has proven this is an unfeature and is
probably causing more harm than good at this stage.

In the future I'll probably make the `dnsbl` block more flexible so that
you can specify your own lists and rules around them.

Index: data/botPolicies.json
--- data/botPolicies.json.orig
+++ data/botPolicies.json
@@ -394,5 +394,5 @@
       "action": "CHALLENGE"
     }
   ],
-  "dnsbl": true
+  "dnsbl": false
 }
